Description: Refer to ssh's Upstart job rather than init script
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: not-needed
Last-Update: 2010-02-28

Index: b/sshd.8
===================================================================
--- a/sshd.8
+++ b/sshd.8
@@ -69,7 +69,7 @@
 .Nm
 listens for connections from clients.
 It is normally started at boot from
-.Pa /etc/init.d/ssh .
+.Pa /etc/init/ssh.conf .
 It forks a new
 daemon for each incoming connection.
 The forked daemons handle
